### PR TITLE
Add several basic tests

### PR DESCRIPTION
**Pull request by marcjansen**
This adds more very basic tests, which at least show how to instanciate instances with minimal configurations.

Additionally this 

* … adds `npm run test:watch` with minimal output, no instrumentation / coverage
* … ensures `CoordinateTransform` instances ca be destroyed
* … fixes the API docs of a config option of the `FeatureGrid`
* … adds a non-critical but annoying TODO to `view/GraphicPool.js`
* … adds three new utility methods to the `TestUtil` helper class:
  * `destroyAll` which helps when cleaning up multiple Ext components
  * `disableLogging` and
  * `enableLogging` to turn logging when testing on and off as desired.

Coverage still is too low IMO, so this is only one step towards reasonable test coverage.

Please review.

### CODE REVIEW

File: karma-watch.conf.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        'https://cdn.jsdelivr.net/gh/highsource/ogc-schemas@2.6.1/scripts/lib/Filter_1_0_0.js',
+        'https://cdn.jsdelivr.net/gh/highsource/ogc-schemas@2.6.1/scripts/lib/GML_2_1_2.js',
+        'https://cdn.jsdelivr.net/gh/highsource/w3c-schemas@1.4.0/scripts/lib/XLink_1_0.js',
+        'https://cdn.jsdelivr.net/npm/proj4@2.5.0/dist/proj4-src.min.js',
+        'src/**/*js',
+        'test/test-helper-functions.js',
+        'test/**/*js',
+        {pattern: 'test/**/*gif', watched: false, included: false, served: true},
+        {pattern: 'test/**/*json', watched: false, included: false, served: true}
+    ];
```
Comment (dnlkoch) [inline karma-watch.conf.js]: Please move to `karma.config.common.js` (or similar) as we could share this config with the `karma.conf`. There might be some other config options to share as well (e.g. `proxies`).

File: test/spec/view/combo/Language.test.js [hunk 2]
```diff
@@ -3,10 +3,19 @@ Ext.Loader.syncRequire([
 ]);
 
 describe('BasiGX.view.combo.Language', function() {
+    var Cls = BasiGX.view.combo.Language;
     var combo;
+    var originalSuccessFn = Cls.prototype.onLoadAppLocaleSuccess;
+    var silentOnLoadAppLocaleSuccess = function() {
+        TestUtil.disableLogging(); // turn annoying loggs temporarily off
```
Comment (dnlkoch) [inline test/spec/view/combo/Language.test.js]: Minor: `loggs` -> `logs`.

File: karma-watch.conf.js (hunk 1 above)
Comment (marcjansen) [reply karma-watch.conf.js]: Done in https://github.com/terrestris/BasiGX/pull/377/commits/c4c51f9aa7784f8e85875e8320a9f3af12df1bde

File: test/spec/view/combo/Language.test.js (hunk 2 above)
Comment (marcjansen) [reply test/spec/view/combo/Language.test.js]: Done in https://github.com/terrestris/BasiGX/pull/377/commits/522564246bd13dc47624a8c5f0ad083ab26c04e8

**Comment by marcjansen** [comment]
Thanks for the review and comments @dnlkoch, I have adressed what you have said and will merge once this is green. Thanks again.

**Pull request merged**